site2/docs/admin-api-overview.md

```diff
@@ -88,3 +83,8 @@ PulsarAdmin admin = PulsarAdmin.builder()
 .build();
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
+
+## How to define Pulsar resource names when running Pulsar in Kubernetes
```

Comment:
Why add this content to this file rather than K8S related sections?

Reply:
That's a very good question.
In our current docs, we have different information on K8S, how to get started with K8S,  how to install, deploy, upgrade and so on. It's most from the operation side. 
Then who cares how to define their Pulsar resource names? It's usually the developers/admin. And they have three ways to define them (REST API, pulsar-admin, client). Since the situation apply to different resources like namespace, tenant, topics..., so it's better to add it in the overview. If they do not know the rule, it's the best place for them to check it after admin setup.